target/linux/ramips/Makefile

```diff
@@ -14,6 +14,7 @@ FEATURES:=squashfs gpio
 MAINTAINER:=John Crispin <john@phrozen.org>
 
 KERNEL_PATCHVER:=4.14
+KERNEL_TESTING_PATCHVER:=4.19
```

Comment:
I would prefer switching to 4.19 directly once it gets merged.

Reply:
It still needs more work.

Reply:
I haven't runtested it yet, but from your PR description it seems like MMC is not working. I don't see any negative comments either.